### DISCUSSION TITLE
add support for input[type='range'] and input[type='toggle']

### DISCUSSION THREAD

**Discussion by nskley** [tailwindlabs/tailwindcss#14409] [Ideas]
Hello, we are currently using the tailwindcss-forms plugin in our project, the only thing missing for us is that we cannot the set the color of inputs of type range and toggle. Would it be possible to add support for those elements ?

**Comment by mrleblanc101** [comment]
`<input type="toggle" />` does not exist 🤔 
Do you mean `<input type="checkbox" switch />` ? 
This can already be styled with the `accent-color` property.
